Content.Client/Zombies/ZombieSystem.cs

```diff
@@ -40,6 +41,7 @@ private void GetInitialInfectedIcon(Entity<InitialInfectedComponent> ent, ref Ge
 
     private void OnStartup(EntityUid uid, ZombieComponent component, ComponentStartup args)
     {
+        if (TryComp<CollectiveMindComponent>(uid, out var collectiv)) { _collectiveMindUpdateSystem.UpdateCollectiveMind(uid, collectiv); }  // Moffstation - Zombies not getting added to their Hivemind
```

Comment:
```suggestion
        // Moffstation - Begin - Update to give zombies their collective mind communication
        if (TryComp<CollectiveMindComponent>(uid, out var collective))
        {
            _collectiveMindUpdateSystem.UpdateCollectiveMind(uid, collective);
        }
        // Moffstation - End
```
sozz I was imprecise in my suggestion.